script/validate-gofmt

```diff
@@ -3,7 +3,7 @@
 source "$(dirname "$BASH_SOURCE")/.validate"
 
 IFS=$'\n'
-files=( $(validate_diff --diff-filter=ACMR --name-only -- '*.go' | grep -v '^\(integration/\)\?vendor/' || true) )
+files=( $(validate_diff --diff-filter=ACMR --name-only -- '*.go' | grep -v '^\(integration/\)\?vendor/\|autogen' || true) )
```

Comment:
Did you verify the pipe operator works (that is, it really ignores the autogen file while not breaking the vendor things)?

Last time I tried, I had quite some issues, which is why I'm asking.